.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -20,32 +20,17 @@ A clear and concise description of what you expected to happen.
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
+**Server details:**
+- Nextcloud version: [e.g. 26.0.1]
+- PHP Version: [e.g. 8.2]
+- Database: [e.g. MariaDB 10.6]
+
 **Client details:**
  - OS: [e.g. iOS]
  - Browser: [e.g. chrome, safari]
  - Version: [e.g. 22]

```

Comment:
The issue I opened was specifically about this line. Can you updated to either say "browser version" or "os version"? I think it's still confusing.

Reply:
Ah thanks, will clarify :)